main.tf

```
@@ -113,7 +115,7 @@ resource "aws_db_parameter_group" "default" {
 }
 
 resource "aws_db_option_group" "default" {
-  count = length(var.option_group_name) == 0 && module.this.enabled ? 1 : 0
+  count = ! local.is_postgres && length(var.option_group_name) == 0 && module.this.enabled ? 1 : 0
```

Could we create a `use_option_group` local and reuse it here and above ? I think it would be more readable than `is_postgres`

```suggestion
  count = ! local.is_postgres && length(var.option_group_name) == 0 && module.this.enabled ? 1 : 0
```